lib/ansible/plugins/action/reboot.py

```diff
@@ -150,25 +200,33 @@ def run_test_command(self, **kwargs):
             raise
 
         result = {}
+
         if command_result['rc'] != 0:
+            msg = 'test command failed: {err} {out}'.format(
+                err=to_native(command_result['stderr']),
+                out=to_native(command_result['stdout']))
             result['failed'] = True
-            result['msg'] = 'test command failed: %s %s' % (to_native(command_result['stderr'], to_native(command_result['stdout'])))
+            result['msg'] = msg
+            raise Exception(msg)
```

Comment:
Please raise some semantically correct exception, like
```suggestion
            raise RuntimeError(msg)
```

Also, args are messed up: you construct `result` dict in this if-branch but not use it at all?

Reply:
I used more appropriate exceptions when raising.

Reply:
I noticed this yesterday, too. It seems I was originally planning to return the output of the test command and do something with it, but never did. Therefore there's no need to store any of this.